### PR TITLE
[SHELL32] Add IDD_LINK_PROBLEM and IDS_LINK_INVALID

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

This PR provides a resource dialog and a resource string for shortcut problem.
JIRA issue: [CORE-16816](https://jira.reactos.org/browse/CORE-16816)

### CODE REVIEW

**Comment by katahiromz** [comment]
無題

File: dll/win32/shell32/lang/fr-FR.rc [hunk 1]
```diff
@@ -709,6 +709,20 @@ BEGIN
     PUSHBUTTON "Modifier l'&icône...", IDC_FOLDERCUST_CHANGE_ICON, 15, 210, 75, 15
 END
 
+IDD_LINK_PROBLEM DIALOGEX 0, 0, 205, 140
```
Comment (KyleKatarn31) [inline dll/win32/shell32/lang/fr-FR.rc:712]: Do you want me to provide FR-fr translation or do you prefer a dedicated PR ?

**Comment by KyleKatarn31** [comment]
Correct me if i'm wrong but, while I clearly see the new ressources (IDD_LINK_PROBLEM ...) I still miss the functional code that uses it. Is it just a draft PR ?

**Comment by Mas4hmad** [comment]
i give the dialog from windows 2003 for comparison.
shell32 (3037)

and the position code of it if you will

File: dll/win32/shell32/lang/fr-FR.rc (hunk 1 above)
Comment (katahiromz) [reply dll/win32/shell32/lang/fr-FR.rc:712]: Translation is later...

**Comment by learn-more** [comment]
> 
> 
> i give the dialog from windows 2003 for comparison.
> shell32 (3037)
> 
> and the position code of it if you will

We don't need to be pixel perfect.

**Comment by binarymaster** [comment]
Is `IDS_LINK_INVALID` used somewhere?

While doing a [grep](https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=IDS_LINK_INVALID) I can see only its definition and translations, but it seems not used anywhere else so far.
